src/qt/rpcconsole.cpp

```diff
@@ -731,8 +731,12 @@ void RPCConsole::clear(bool clearHistory)
         );
 
     message(CMD_REPLY, (tr("Welcome to the %1 RPC console.").arg(tr(PACKAGE_NAME)) + "<br>" +
-                        tr("Use up and down arrows to navigate history, and <b>Ctrl-L</b> to clear screen.") + "<br>" +
-                        tr("Type <b>help</b> for an overview of available commands.")) +
+#ifdef Q_OS_MAC
+                        tr("Use up and down arrows to navigate history, and <b>⌘cmd-L</b> to clear screen.") + "<br>" +
+#else
+                        tr("Use up and down arrows to navigate history, and <b>Ctrl-L</b> to clear screen.") + "<br>" +		
+#endif
+			tr("Type <b>help</b> for an overview of available commands.")) +
```

Comment:
If we are already bikeshedding:
Alternative suggestion would be adding `QString generalFunctionKey()` to platformstyles.cpp.
Then use something like `tr("Use up and down arrows to navigate history, and %1-L to clear screen.").arg(platformStyle->generalFunctionKey())`